index.js

```diff
@@ -100,7 +104,7 @@ function Angular2ConventionsLoader(source, sourcemap) {
           return 'template:' + replaceStringsWithRequires(url);
         })
     }
-    if (decorator === 'Component' && templateUrlRegex.test(metadata)) {
+    if (decorator === 'Component' && styleUrlsRegex.test(metadata)) {
```

Comment:
I thought this was done by mistake. Because here we need to check if there are style urls and if so, update them.